markdown/articles/2022/02/09/axionmegareport.md

```diff
@@ -0,0 +1,575 @@
+on some of the fairer strategies. Something like Elves would have likely been a decent choice
+for the event. 
+
+:::row{variant=centered}
+- Thalia, Guardian of Thraben
+- Delver of Secrets
+- Cloudpost
+- Exploration
+:::
+
+Righty, now for the Doomsday bit :3 
+
+## Winner's Event Report :)
+
+### Decklist
+
+I was one of the lucky few who managed to secure a ticket and with only the knowledge that 
+"there will probably be UR Delver and Death & Taxes" about I had to try and come up with my
+list. To make things slightly trickier I had gone 1-3 at the last local FNM I had played in
+(making me feel a little bummed out) and, probably more importantly, the :card[Ragavan, Nimbler Pilferer]
+ban had come into effect so I was probably due to make some changes to my [previous list](/articles/2022/01/22/zero-respect#the-decklist)
+whilst being unable to test for the event at all thanks to general life things getting in the way.
+
+As a result I took my previous list as a starter and begun conversing online with various community
+members to try and assess what changes I might want to make (shout out to Otterlady, Nevilshute and
+Prepare4Robots especially on this). I knew I wanted to add additional discard effects into the
+list somewhere and was most worried about Blue based "Soup" decks (Bant, 4C, Jeskai Breacher) just
+because they can run such a diverse spread of relevant hate that, in normal lists, requires answers
+that are otherwise really bad in those matchups. I spent a lot of time agonising over trying an
+alternative way to bypass a :card[Torpor Orb] from say a RUG Saga deck such as good old :card[Tendrils of Agony]
+or ways to remove a variety of other nuisance elements like :card[Meddling Mage] as well with tools
+like :card[Kolaghan's Command] however it was decided to just keep to the theme of what I know and
+just not respect that sort of behaviour. If blue decks want to bring such rude toys to play then
+they are welcome to it!
+
+The final major talking point which I was convinced to try was removing :card[DA] from the list.
+A lot of people have started to run the deck still with :card[Con] but without Analysis as the
+former still has [relevant pile application](/chapters/meandeck/brainstorm.md#brainstorm-pile---brainstorm-and-free-cantrip-available)
+and acts as a decent cantrip to clear 2 cards in the face of a :card[Narset, Parter of Veils] 
+whilst the latter only enables specific fast/efficient piles but ones that are very susceptible 
+to a lot of hate in the format. Thanks to recent trends and discussion with folks I decided to 
+risk it by removing the card from the deck entirely (it was discussed putting it in the board but
+it shines most of all in game 1 scenarios).
+
+When Consider was first spoilt I had in fact said the following:
+
+> I'm going to buy Consider. Buy DA. Then probably only end up on one consider no  DA >.<
+> - Doishy 08/06/2021
```

Comment:
Add a blank line (with `>` prefix) to make a new paragraph. Alternatively, add 2 space characters at the end of the first line to insert a line break (no paragraph)